hack/common.sh

```diff
@@ -41,6 +41,9 @@ FULL_IMAGE_NAME="${IMAGE_REGISTRY}/${REGISTRY_NAMESPACE}/${IMAGE_NAME}:${IMAGE_T
 default_local_storage_image="quay.io/gnufied/local-registry:v4.2.0"
 LOCAL_STORAGE_IMAGE_NAME="${LOCAL_STORAGE_IMAGE:-$default_local_storage_image}"
 
+UPGRADE_TO_OCS_REGISTRY_IMAGE="${UPGRADE_TO_OCS_REGISTRY_IMAGE:-}"
```

Comment:
I bet we could use the ocs-operator nightly here as the secondary images. @kshlm @jarrpa any progress on the nightly build? Are we tracking that?

Reply:
https://github.com/openshift/ocs-operator/issues/75 describes an alternative to a nightly.  We'd re-build the container images on each merged pr as a post submit job.

Reply:
The secondary image that we want to upgrade from needs to have a different version than the current image, and also need "replaces" attribute in the csv. And both of the versions need to be specified in their respective channels in the package.yaml. Once we create that secondary image with the right changes, we can use that as a starting point for upgrades.

I will send a separate pr with the csv changes we need.